frontend/pages/queries/edit/components/EditQueryForm/EditQueryForm.tsx

```diff
@@ -638,12 +638,32 @@ const EditQueryForm = ({
     "differential_ignore_removals",
   ].includes(lastEditedQueryLoggingType);
 
+  // Note: The backend is not resetting the query reports with equivalent platform strings
```

Comment:
This was the best way I could think of to test equivalence, especially if we ever add new platforms. @jacobshandling would love your input on the naming of this equivalence checker.

Reply:
LGTM